common/catalog/docker/docker.bom

```diff
@@ -271,14 +269,12 @@ brooklyn.catalog:
           brooklyn.config:
             uniqueTag: docker-bind-url-generator
             enricher.suppressDuplicates: false
-            enricher.triggerSensors:
-              - $brooklyn:sensor("docker.port")
```

Comment:
OK, this is incorrect - a `Transformer` will execute when it is first created, so this will run and use the two config key values. It just won't run again, but this is not required.